File: TUTORized/Models/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace TUTORized.Models
{
    public class Message
    {
        [JsonProperty("id")]
        public string Id { get; set; }

        [JsonProperty("fromUser")]
        public User FromUser { get; set; }

        [JsonProperty("toUser")]
        public User ToUser { get; set; }

        [JsonProperty("messageBody")]
        public string MessageBody { get; set; }
    }
}

File: TUTORized/ClientApp/models/Message.ts
import { User } from "./User";

export class Message {
    id: string;
    fromUser: User;
    toUser: User;
    messageBody: string;

    constructor() {
        this.id = "";
        this.fromUser = new User();
        this.toUser = new User();
        this.messageBody = "";
    }
}
